Bottom: c8f4da85137abaf16c3258190a38f7fd4656e647
Top:    f0df1861b2edbb4a1d591b4b5c78b5a4f463bd5d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-25 13:23:48 +0100

Refresh of demo-of-exporter-specific-expo

---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index 1eb1e2a..676a6fe 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -420,6 +420,7 @@ gerber_expose (HID * hid, BoxType *drawn_area, void *item)
   int i;
   int group;
   int nplated, nunplated;
+  extern bool Gathering;
 
   HID *old_gui = gui;
   hidGC savebg = Output.bgGC;
@@ -437,6 +438,8 @@ gerber_expose (HID * hid, BoxType *drawn_area, void *item)
   PCB->Data->SILKLAYER.Color = PCB->ElementColor;
   PCB->Data->BACKSILKLAYER.Color = PCB->InvisibleObjectsColor;
 
+  Gathering = false;
+
   memset (print_group, 0, sizeof (print_group));
   for (i = 0; i < max_copper_layer; i++)
     {
@@ -494,6 +497,8 @@ gerber_expose (HID * hid, BoxType *drawn_area, void *item)
   Output.fgGC = savefg;
   Output.bgGC = savebg;
   Output.pmGC = savepm;
+
+  Gathering = true;
 }
 
 static void
